src/Chapter2.hs

```diff
@@ -741,7 +751,16 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains _ [] = []
+contains n (x:xs) 
+  | (isThreeIn n x) /= [] = (isThreeIn n x):(contains n xs)
+  | otherwise = contains n xs
+  where
+    isThreeIn :: Int -> [Int] -> [Int]
+    isThreeIn m y
+      | (length (filter (==m) y))>0 = y
+      | otherwise = []
```

Comment:
There is a standard function [`elem`](https://hackage.haskell.org/package/base-4.15.0.0/docs/Prelude.html#v:elem) that does this check for you 🙂